File: exampleadapters/src/main/java/foo/bar/example/asafadapters/ui/PlaylistsFragment.java
package foo.bar.example.asafadapters.ui;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import foo.bar.example.asafadapters.R;


public class PlaylistsFragment extends Fragment {

    public static PlaylistsFragment newInstance() {
        PlaylistsFragment fragment = new PlaylistsFragment();
        return fragment;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_playlists, null);
    }

}

File: example-kt-03adapters/src/main/java/foo/bar/example/foreadapterskt/ui/playlist/PlaylistsFragment.kt
package foo.bar.example.foreadapterskt.ui.playlist

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import foo.bar.example.foreadapterskt.R


class PlaylistsFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_playlists, null)
    }

    companion object {

        fun newInstance(): PlaylistsFragment {
            return PlaylistsFragment()
        }
    }

}
